napalm_ios/ios.py

```diff
@@ -78,6 +79,23 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         self.config_replace = False
         self.interface_map = {}
 
+    def file_copy(self, src, dest=None, file_system='flash:'):
+        """
+        SCP file to device filesystem, filesystem defaults to flash:.
+
+        Return None or raise exception
+        """
+        if dest is None:
+            dest = basename(src) 
+        if src:
+            (return_status, msg) = self.scp_file(source_file=src,
```

Comment:
scp_file is in ios.py and largely uses underlying Netmiko file transfer method. 

I would think we would want both a put operation and a get operation.